dataactbroker/handlers/fileHandler.py

```diff
@@ -859,31 +871,39 @@ def getProtectedFiles(self):
         response["urls"] = self.s3manager.getFileUrls(bucket_name=CONFIG_BROKER["static_files_bucket"], path=CONFIG_BROKER["help_files_path"])
         return JsonResponse.create(StatusCode.OK, response)
 
-    def completeGeneration(self, generationId):
+    def completeGeneration(self, generationId, file_type=None):
```

Comment:
It looks like a good chunk of this method is specific to processing the remote file. What do you think about splitting this apart and only calling the bits relevant to local generation from `addJobInfoForDFile`?